sdk/keyvault/azure-security-keyvault-jca/pom.xml

```diff
@@ -133,13 +134,20 @@
         <version>3.5.2</version> <!-- {x-version-update;org.apache.maven.plugins:maven-shade-plugin;external_dependency} -->
         <executions>
           <execution>
+            <id>minimize-shade</id>
             <goals>
               <goal>shade</goal>
             </goals>
             <configuration>
               <shadeTestJar>false</shadeTestJar>
               <createSourcesJar>${createSourcesJar}</createSourcesJar>
               <shadeSourcesContent>true</shadeSourcesContent>
+              <minimizeJar>true</minimizeJar>
+              <artifactSet>
+                <includes>
+                  <include>org.bouncycastle:*</include>
+                </includes>
+              </artifactSet>
```

Comment:
I think @g2vinay meant that we should be copying relevant source from the library instead of shading the jar.

Reply:
Yes, I tried the copying, the difficulty of this approach is to understand the BouncyCastle code and replace it. One class refer to another class, finally, there are many references and some classes are relatively large. I would like use the shading by the shade plugin.